docusaurus/docs/backends/03-casper.md

```diff
@@ -22,7 +22,6 @@ The other exposed types are:
 
 * `CallArgs` - wraps around casper's [`RuntimeArgs`][runtime_args];
```

Comment:
No `CallArgs` in the new version, we've got `CallDef`, but it is not just a wrapper, holds a more fields